sparkmagic/sparkmagic/magics/sparkmagicsbase.py

```diff
@@ -34,6 +37,21 @@ def __init__(self, shell, data=None, spark_events=None):
             spark_events = SparkEvents()
         spark_events.emit_library_loaded_event()
 
+    def execute_spark(self, cell, output_var, samplemethod, maxrows, samplefraction, session_name):
+        (success, out) = self.spark_controller.run_command(Command(cell), session_name)
+        if not success:
+            self.ipython_display.send_error(out)
+        else:
+            self.ipython_display.write(out)
+            if output_var is not None:
+                spark_store_command = self._spark_store_command(cell, output_var, samplemethod, maxrows, samplefraction)
+                df = self.spark_controller.run_command(spark_store_command, session_name)
```

Comment:
As it is, we would simply be catching the exception and displaying it, but I wonder if there's a possibility for a better error message... Ideas?